docs/products/clickhouse/howto/use-play.rst

```diff
@@ -0,0 +1,14 @@
+Use ClickHouse® play UI
+========================
+
+ClickHouse® includes a built-in user interface for running SQL queries. You can access it through HTTPS protocol. Follow these steps to use it:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your ClickHouse service.
```

Comment:
Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse® service.

Reply:
Note to the author: I'm not sure if we need to mention the full form Aiven for ClickHouse® every time or mentioning it once at the beginning will suffice. 

Also, "choose the right project" part is a soft suggestion so feel free to disregard it.

Reply:
I still added it. I think it makes sense, when we talk about our service and not ClickHouse open source project